### DISCUSSION TITLE
Select Branch and Workflow not loading

### DISCUSSION THREAD

**Discussion by philliphartin** [codemagic-ci-cd/codemagic-docs#1670] [Q&A] [answered]
<img width="666" alt="CleanShot 2023-05-17 at 17 16 03@2x" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/1385026/8ec2fb9e-8ee5-415e-80ef-7b8751cd9697">

Unable to load branch details or workflow.

It was working seamlessly, then all of a sudden not working. I've tried different browsers, logging in and out etc. I'm assuming service may be impacted?

**Comment by thedavidpeto** [comment]
Duplicate of [1669](https://github.com/orgs/codemagic-ci-cd/discussions/1669)

**Reply by TFJL** [reply]
Yip same here

**Reply by dtrdic** [reply, marked as answer]
can you try it now? should be ok

**Reply by thedavidpeto** [reply]
Yes it works now

**Comment by philliphartin** [comment]
Thanks!
